src/app/events/events.page.scss

```diff
@@ -1,114 +1,115 @@
-.event-container {
-  border-radius: 4px;
-  background: var(--surface-screen-screen-card-black-dark, #0d0d0d);
-  box-shadow: 0px 0px 8px 1px rgba(255, 220, 35, 0.3);
-  padding: 8px;
-  width: 91%;
-  margin: 16px auto;
-  z-index: -1;
-}
-
-h3 {
-  margin: 4px 0;
-  font-size: 16px;
-  font-style: normal;
-  font-weight: 500;
-  line-height: 150%;
-}
-
-.event-description {
-  margin-bottom: 8px;
-  color: #c2bfbe;
-  /* font-family: Open Sans; */
-  font-size: 12px;
-  font-style: normal;
-  font-weight: 400;
-  line-height: 150%;
-}
-.image-wrapper {
-  position: relative;
-}
-.image-wrapper img{
-  display: block;
-  max-width: 100%;
-  height: auto;
-}
-
-.event-image {
-  border-radius: 4px;
-}
-.event-location {
-  color: #0d0d0d;
-  font-family: Open Sans;
-  font-size: 12px;
-  font-style: normal;
-  font-weight: 600;
-  line-height: 150%;
-  display: inline-flex;
-  padding: 0px 4px;
-  align-items: flex-start;
-  gap: 4px;
-  border-radius: 4px;
-  background: #ff7418;
-  position: absolute;
-  bottom: 26px;
-  left: 4px;
-}
-.event-date {
-  display: inline-flex;
-  padding: 0px 4px;
-  align-items: flex-start;
-  gap: 10px;
-  border-radius: 4px;
-  background-color: #ffdc23;
-  color: #0d0d0d;
-  /* font-family: Open Sans; */
-  font-size: 12px;
-  font-style: normal;
-  font-weight: 600;
-  line-height: 150%;
-  position: absolute;
-  bottom: 4px;
-  left: 4px;
-}
-
-.event-like-btn {
-  /* position: absolute;
-  font-size: 30px;
-  color: #f44336;
-  top: 4px;
-  right: 4px; */
-  z-index: 1000;
-}
-.event-like-btn div img{
-  min-width: 28px;
-}
-.event-info {
-  position: relative;
-}
-
-.more-btn {
-  display: flex;
-  width: 47%;
-  padding: 4px 16px;
-  justify-content: center;
-  align-items: center;
-  gap: 10px;
-  border-radius: 4px;
-  background: #ff7418;
-}
-
-.more-btn p {
-  color: #0d0d0d;
-  text-align: center;
-  font-family: Open Sans;
-  font-size: 14px;
-  font-style: normal;
-  font-weight: 600;
-  line-height: 150%;
-}
-
-.location-img {
-  width: 16px;
-  height: 16px;
-}
+.event-container {
+  border-radius: 4px;
+  background: var(--surface-screen-screen-card-black-dark, #0d0d0d);
+  box-shadow: 0px 0px 8px 1px rgba(255, 220, 35, 0.3);
+  padding: 8px;
+  width: 91%;
+  margin: 16px auto;
+  z-index: -1;
+}
+
+h3 {
+  margin: 4px 0;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 150%;
+}
+
+.event-description {

```

Comment:
Do we need this css and below here anymore?

Reply:
No we didnt. Removed.